src/index.js

```diff
@@ -4,8 +4,10 @@
  */
 import React from 'react'
 import {css, styleSheet} from 'glamor'
+import * as jsxStyleComponents from 'glamor/jsxstyle'
```

Comment:
I'd rather not import this from `glamor/jsxstyle` especially considering that glamor is dropping this in favor of `glamorous` in 3.0 ;-) We should construct the components ourselves. And actually it should be fairly simple and straightforward to do that.